feedback/static/html/studio_view.html

```diff
@@ -1,111 +1,99 @@
 <div class="wrapper-comp-settings is-active editor-with-buttons" id="settings-tab">
-  <ul class="list-input settings-list">
-    <li class="field comp-setting-entry is-set"> This XBlock allows
-      you to collect student feedback on pieces of the course. This
-      may be helpful either for course improvement, or to give
-      students a chance to reflect on what they have done.</li>
-  
-    <li class="field comp-setting-entry is-set"> Prior to using this
-      block, we recommend reading about best practices in the edX
-      documentation. In particular, we do recommend asking for
-      structured, specific feedback, and we advise using this in
-      conjunction with
-      the <a href="http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/content_experiments/content_experiments_configure.html">RCT
-      framework</a> for matrix sampling.</li>
+    <ul class="list-input settings-list">
+        <li class="field comp-setting-entry is-set"> This XBlock allows
+        you to collect student feedback on pieces of the course. This
```

Comment:
Do we need to wrap this text to translations string? Here and below.

Reply:
I think yes, but no, when I collect translations of the xblock, django translation toll doesn't collect html translations. Do we need to translate xblock content for staff?